great_tables/_locale.py

```diff
@@ -1,7 +1,14 @@
 from __future__ import annotations
-from typing import Optional
-import pandas as pd
-import pkg_resources
+
+from csv import DictReader
```

Comment:
Replaced all pandas DataFrames with the built in python DictReader. This returns a list of dictionaries (where each dictionary is a row)